packages/api/src/util/ensemble.js

```diff
@@ -45,8 +47,8 @@ if (process.env.NODE_ENV === 'integration') {
   baseUrl = 'http://localhost:' + process.env.PORT;
 }
 
-//Known aliases for units from ensemble mapping to convert-units representation
-const ENSEMBLE_UNITS_MAPPING = {
+// Known aliases for units from ensemble mapping to convert-units representation as needed by the old sensor readings controller for incoming webhook data
+const ENSEMBLE_UNITS_MAPPING_WEBHOOK = {
```

Comment:
This is only used in the "old" code, right? Perhaps we could make that a bit more explicit in the naming by prefacing with "deprecated", "legacy", "v1" or something of the sort